### DISCUSSION TITLE
Episodes not released yet!

### DISCUSSION THREAD

**Discussion by rebatnaath** [pystardust/ani-cli#952] [General] [answered]
To show how great this is, I installed ani-cli on my friend's laptop. But it doesn't work; in the past, a similar error prevented me from being able to search; however, with this issue, I can search and get results; nevertheless, when I pick to play the anime, a warning that says **Episodes not released yet!** appears. Are there any solutions available? I've tried removing and reinstalling the program.

![image](https://user-images.githubusercontent.com/78601591/209422221-21ffdbea-d0a1-40e0-b913-0c7f4c211df2.png)

**Comment by justchokingaround** [comment, marked as answer]
#948
